src/timedata.cpp

```diff
@@ -76,30 +75,24 @@ void AddTimeData(const CNetAddr& ip, int64_t nOffsetSample)
     // So we should hold off on fixing this and clean it up as part of
     // a timing cleanup that strengthens it in a number of other ways.
     //
-    if (vTimeOffsets.size() >= 5 && vTimeOffsets.size() % 2 == 1)
-    {
+    if (vTimeOffsets.size() >= 5 && vTimeOffsets.size() % 2 == 1) {
         int64_t nMedian = vTimeOffsets.median();
         std::vector<int64_t> vSorted = vTimeOffsets.sorted();
         // Only let other nodes change our time by so much
-        if (abs64(nMedian) <= std::max<int64_t>(0, gArgs.GetArg("-maxtimeadjustment", DEFAULT_MAX_TIME_ADJUSTMENT)))
-        {
+        if (abs64(nMedian) <= std::max<int64_t>(0, gArgs.GetArg("-maxtimeadjustment", DEFAULT_MAX_TIME_ADJUSTMENT))) {
             nTimeOffset = nMedian;
-        }
-        else
-        {
+        } else {
             nTimeOffset = 0;
 
             static bool fDone;
-            if (!fDone)
-            {
+            if (!fDone) {
                 // If nobody has a time different than ours but within 5 minutes of ours, give a warning
                 bool fMatch = false;
-                for (const int64_t nOffset : vSorted)
-                    if (nOffset != 0 && abs64(nOffset) < 5 * 60)
-                        fMatch = true;
+                for (const int64_t nOffset : vSorted) {
+                    if (nOffset != 0 && abs64(nOffset) < 5 * 60) fMatch = true;
```

Comment:
I know clang-format did it due to `AllowShortIfStatementsOnASingleLine: true` and so there is nothing to do in this PR, but I just want to mention that putting the `if` condition and body on one line like `if (A) B;`
* makes life in gdb more difficult (if one wants to set a breakpoint on B)
* makes backtraces less usefull (if it crashed on line 92, did it crash in `A` or in `B`?)
* skews line based code coverage (if just `A` is executed and it was `false` then the code coverage may show the entire line 92 covered, giving the false impression that `B` was also executed).

Reply:
Lean to agree with you. Just want to preserve ACKs :)

Reply:
We only allow trivial statements in single line ifs, so it should always be clear whether it crashed in A or B.

Also, coverage reports reports generally include branch coverage.